cardano-ledger-core/src/Cardano/Ledger/Hashes.hs

```diff
@@ -59,6 +62,10 @@ data EraIndependentScriptData
 
 data EraIndependentPParamView
 
+data EraIndependentScriptIntegrity
+
+{-# DEPRECATED EraIndependentWitnessPPData "Use EraIndependentScriptIntegrity instead" #-}
```

Comment:
We should make this a type alias, surely? We can certainly alias empty data types.

Reply:
yes indeed, thanks!